test/init_test.sh

```diff
@@ -12,8 +12,8 @@ if [ "$(whoami)" != "$TEST_USER" ]; then
     exit 0
 fi
 
-GO_VERSION="1.22.0"
-NERDCTL_VERSION="2.0.0-beta.3"
+GO_VERSION="1.23.2"
+NERDCTL_VERSION="2.0.0-rc.3"
```

Comment:
Now GA

Reply:
Fixed.
Congratulations on nerdctl 2.0.0 GA! 🎉

Reply:
The fix doesn't seem pushed?

Reply:
I fixed the wrong commit.